front/src/components/stepper/stepper.component.css

```diff
@@ -0,0 +1,70 @@
+.stepper {
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  position: relative;
+}
+
+  .stepper .step {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    flex-basis: 200px;
+    padding: 24px;
+  }
+
+    .stepper .step .title {
+      font-size: 18px;
+    }
+
+    .stepper .step .sub-title {
+      font-size: 14px;
+    }
+
+    .stepper .step .icon {
+      text-decoration: none;
+      margin: 30px 0 15px;
+      border-radius: 50%;
+      height: 45px;
+      width: 45px;
+      line-height: 45px;
+      text-align: center;
+      z-index: 999;
+    }
+
+  .stepper::after {
+    position: absolute;
+    top: 50%;
+    left: 10%;
+    height: 3px;
+    width: 80%;
+    background-color: #DDDDDD;
+    content: "";
+  }
+
+  .stepper::before {
+    position: absolute;
```

Comment:
why do you choose absolute position and centering via top left instead of flex possibilities?
also it is better not to use width and height in percents measuring